packages/eslint-config-turbo/flat/index.ts

```diff
@@ -0,0 +1,16 @@
+import plugin from "eslint-plugin-turbo";
+import {} from "eslint";
+
+const thing = [
```

Comment:
Can we get away with `export default = [...` instead of the variable? If not can we rename it to something better than `thing`?

Reply:
Agree, let's use a better name!

Reply:
Haha, I was trying to make an error go away, my bad.